confluence-syntax-xhtml/src/test/resources/confluence+xhtml10/specific/link/link11.test

```diff
@@ -9,11 +9,8 @@
 .#-----------------------------------------------------
 beginDocument
 beginParagraph
-beginLink [Typed = [true] Type = [user] Reference = [user] Parameters = [[anchor] = [anchor]]] [false]
-onWord [label]
-endLink [Typed = [true] Type = [user] Reference = [user] Parameters = [[anchor] = [anchor]]] [false]
+onMacroInline [mention] [reference=user]
```

Comment:
Losing the label and the anchor sounds like a regression.

Reply:
I couldn't find a structure in Confluence that would look like this
```
<ac:link ac:anchor="anchor">
  <ri:user ri:username="user" />
  <ac:plain-text-link-body><![CDATA[label]]></ac:plain-text-link-body>
</ac:link>
```
and I couldn't create it either. 

When it comes to the `<ri:user>` tag, it is either inside a link tag, which means it represents a mention:
`<ac:link><ri:user ri:account-id="someId" /></ac:link>`
or inside a parameter tag for some macro:
`<ac:parameter ac:name="user"><ri:user ri:account-id="someId" /></ac:parameter>`

I think we should drop this test, unless it was possible to get the said structure in older versions of confluence. 🤔

Reply:
> unless it was possible to get the said structure in older versions of confluence

Honestly, I don't know.

Reply:
OK, anyway, I doubt it's an important use case. Let's say it's OK for now, and we'll see if someone complains.